Description: Fix two typos in acct_gather.conf.5 man page
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=11627
Last-Update: 2021-05-16

--- slurm-wlm-20.11.7.orig/doc/man/man5/acct_gather.conf.5
+++ slurm-wlm-20.11.7/doc/man/man5/acct_gather.conf.5
@@ -12,7 +12,7 @@ DEFAULT_SLURM_CONF parameter or at execu
 environment variable. The file will always be located in the
 same directory as the \fBslurm.conf\fP file.
 .LP
-Parameter names are case insensitive but parameter values are case sensistive.
+Parameter names are case insensitive but parameter values are case sensitive.
 Any text following a "#" in the configuration file is treated
 as a comment through the end of that line.
 The size of each line in the file is limited to 1024 characters.
@@ -159,7 +159,7 @@ The InfluxDB plugin provides the same in
 instead send information to the configured InfluxDB server.
 .P
 The InfluxDB plugin is designed against 1.x protocol of InfluxDB. Any site
-running a v2.x InfluxDB server will need to configure a v1.x compatiblity
+running a v2.x InfluxDB server will need to configure a v1.x compatibility
 endpoint along with the correct user and password authorization. Token
 authentication is not currently supported.
 .SS
